### ISSUE TITLE
Jupyter Server Initialization Failed

### ISSUE THREAD

**Issue by ctvarlan** [jupyterlab/jupyterlab-desktop#464]
Jupyter Server Initialization Failed
Jupyter Server, which is a prerequisite for JupyterLab Desktop, did not initialize properly.
This might be because of an issue with the selected environment. You can change environment to try a different server using buttons below.
If this does not seem right, please report this issue in the JupyterLab Desktop repository, providing the error details presented below:
Error: spawn cmd.exe ENOENT

That is the message I get when I try to open Jupyter Lab Desktop.
If I follow with the "Change Environment" button then it pops this dialog window with two options:

 **Set Python Environment
 Select the Python executable in the conda or virtualenv environment you would like to use for JupyterLab Desktop. Python packages in the environment selected need to meet the following requirements: jupyterlab >=3.1.0. Prebuilt extensions installed in the selected environment will also be available in JupyterLab Desktop.

[ ]Use the bundled Python environment
[ ]Use a custom Python environment
C:\Users\<user>\AppData\Roaming\jupyterlab-desktop\jlab_server\python.exe
Select Python path
Apply and restart
Cancel**

For either option I choose I land on the same message with "Error: spawn cmd.exe ENOENT"

**Comment by orbanbalage** [comment]
I seem to have the same issue on Mac. I'm running a fresh install.

```Error: Jupyter Server process terminated before the initialization completed```

```
➜  ~ jlab --version
3.4.6-1

➜  ~ sw_vers
ProductName:		macOS
ProductVersion:		13.0
BuildVersion:		22A380
```

[main.log](https://github.com/jupyterlab/jupyterlab-desktop/files/9889056/main.log)

**Comment by tos-kamiya** [comment]
I had a similar symptom on a Windows 10 PC and investigated the cause and found that the environment variable PATH was set to the wrong value and cmd.exe could not start.
I found a way by searching the web for keywords like "reset environment variables in Windows 10" and was able to get it installed.

However, this solution is (or at least, seems) not guaranteed by any authorities and, of course, might have a negative impact on already installed applications, so it is better to use it as a last resort.

**Comment by mbektas** [comment]
@tos-kamiya did you run into issues when launching from command prompt or UI or both?

can you describe what was wrong with the PATH? we include the desktop app process' environment variables when launching the server.

by the way, have you tried the latest version 3.6.1-1?

**Comment by tos-kamiya** [comment]
On one Windows 10 PC, I attempted to install JupyterLab Destkop 3.6.1-1.
1. ran JupyterLab-Setup-Windows.exe and finished JupyterLab setup.
2. When I tried to run JupyterLab (GUI app), I got a message "Python envrionment not found. Install using the bundled installer or Change the default Python environment. Install using the bundled installer or Change the default Python environment.
3. When I clicked on "Install using the bundled installer," I got the error message "Error: spawn cmd.exe ENOENT. If it had executed successfully, the dialog "Welcome to JupyterLabDesktopAppServer 3.6.1-1(64-bit) Setup" would have appeared. (Actually, this is what was displayed after troubleshooting was complete.)
4. After some investigation, I found that the environment variables on that PC were strange: the directory where cmd.exe is located (C:\Windows\System32) was not stored in the path variable.
5. By resetting the Windows PC environment variables, I successfully proceeded to install JupyterLabDektopAppServer.

I hope this information is helpful.
